config/install/image.style.breakpoint_lg_1x.yml

```diff
@@ -9,6 +11,6 @@ effects:
     id: image_scale
     weight: 1
     data:
-      width: 1024
+      width: 992
       height: null
-      upscale: false
+      upscale: true
```

Comment:
Upscaling? :grimacing: That has never been a good thing IMO

Reply:
Desired? No. Problem solving? Somewhat.

This is open for discussion as it does create grainy images but with the especially large 2X 2XL breakpoint (3000px wide) this solves a problem of not having enough pixels when a user inputs a small image. The upscaling will likely have a mitigated effect as image will only display to the max size of the screen and the browser does a good job of downsizing on the smaller screen sizes. Do we need to pull in the other devs and have a quick mind meld on this?

Reply:
yeah lets have a working huddle. Maybe throw some different images at it too.